integration_tests/pom.xml

```diff
@@ -56,6 +56,12 @@
                 <spark.test.version>3.1.1-SNAPSHOT</spark.test.version>
             </properties>
         </profile>
+        <profile>
+            <id>spark320tests</id>
+            <properties>
+                <spark.test.version>3.2.0-SNAPSHOT</spark.test.version>
```

Comment:
Shouldn't this be using `${spark320.version}` from the parent POM?

Reply:
I had it initially like that but then decided to leave it consistent with the rest of the profiles in this pom.

Reply:
I think this (and the other profiles) should be fixed, otherwise updating the property on the command-line doesn't do the right thing here.

Reply:
Another change I've been eager to make is to replace most of the 2.12 mentions by `scala.binary.version`

Reply:
At one point we've replaced most of the ones that we can.  Maven squawked loudly when we tried putting the top-pom variable into the project names.  Maybe some more references have creeped in since I last looked at this.

Reply:
I think maven squawks only about using references inside `<parent>` :)

Reply:
No it wasn't that, obviously.  You can't expand a variable before it's defined.